src/compiler/scala/tools/nsc/symtab/classfile/ClassfileParser.scala

```diff
@@ -904,6 +904,11 @@ abstract class ClassfileParser(reader: ReusableInstance[ReusableDataReader]) {
           }
           in.skip(attrLen)
 
+        case tpnme.PermittedSubclassesATTR =>
+          debuglog(s"$sym is Java sealed.")
+          sym.setFlag(SEALED)
+          in.skip(attrLen) // https://openjdk.java.net/jeps/360 but ignore permits and respect only Scala semantics
```

Comment:
IIUC, this sets `sealed` but doesn't set the symbol's `children` right? Can this lead to false exhaustivity warnings / reachability errors in pattern matching?